app/components/mandatarissen/mandataris-card.hbs

```diff
@@ -103,24 +103,43 @@
         </div>
       </div>
     {{/if}}
-    {{#if (await this.vervangersDoor)}}
-      <div class="au-o-grid__item au-u-5-6">
+    {{#if (await @mandataris.status.isVerhinderd)}}
+      {{#if
+        (not (or (await this.vervangersVan) @mandataris.bekleedt.isSchepen))
+      }}
+        <div class="au-o-grid__item au-u-1-2">
+          <div class="au-c-description-label"></div>
+          <div class="au-c-description-value au-u-margin-top-tiny">
+          </div>
+        </div>
+      {{/if}}
```

Comment:
Fixes the haywire element

 GR: 
![image](https://github.com/user-attachments/assets/6565c189-770e-432e-bf8c-3b19f904caf5)

GR met dubbele vervanger:
![image](https://github.com/user-attachments/assets/a09c2ad5-0d86-49ca-818c-83ce71f983f9)


Schepenen:
![image](https://github.com/user-attachments/assets/ae1c0442-77fd-4380-995c-f746e8444e8c)

Schepenen met dubbele vervanger:
![image](https://github.com/user-attachments/assets/9af58f60-3214-483a-a304-b019ad9b1685)